builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -347,8 +348,13 @@ node() {
               }              
 
               def stackResources = utilModule.getStackResources(stackName, config['region'], credsId)
-              def lambdaArns = utilModule.createAllStackResources(whiteListModule, events, config, stackResources, environment_logical_id)
-               lambdaArnInfo = getLambdaARN(lambdaArns);
+              arnsMap = utilModule.createAllStackResources(whiteListModule, events, config, stackResources, environment_logical_id)
+              lambdaArnInfo = getLambdaARN(arnsMap['lambda']);
+
+              if (config['iamRoleARN']) {
+                def assets_api = g_base_url + "/jazz/assets"
+                utilModule.archiveCustomRole(assets_api, auth_token, config, environment_logical_id, events)
```

Comment:
When user providers role, we archive the Jazz created role (calling that `CustomRole`)?

Reply:
yes